lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -171,9 +173,29 @@ class VersionSectionSelector extends Component<SelectorProps> {
       })
     }
 
-    const text = hasCriticalError
+    // Do the same for the mobility data results
+    let highestMBPriority = 'UNKNOWN'
+    if (version && version.mobilityDataResult) {
+      version.mobilityDataResult.notices.forEach(notice => {
+        const level = errorPriorityLevels[mobilityDataValidationErrorMapping[notice.severity]]
+        if (level < errorPriorityLevels[highestMBPriority]) {
+          highestMBPriority = mobilityDataValidationErrorMapping[notice.severity]
+        }
+      })
+    }
+
+    const dtValidationCount = hasCriticalError
       ? 'critical error'
-      : validationSummary.errorCount
+      : numeral(validationSummary.errorCount).format('0a')
+
+    const mbValidationCount =
+      version.mobilityDataResult &&
+      version.mobilityDataResult.notices &&
```

Comment:
do we... not have null coalescing operator in datatools? I thought we did.

Reply:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/92878779/228421393-ab916439-2273-4344-bed5-9fb6a819d8b6.png">
huh

Reply:
weeping